codespell_lib/data/dictionary.txt

```diff
@@ -11525,6 +11526,8 @@ finess->finesse
 finialization->finalization
 finializing->finalizing
 finilizes->finalizes
+finisch->finish, finish,
```

Comment:
The word `finish` is mentioned twice. Did you mean `Finnish` in one of the cases (language in Finland)?

Reply:
This was caught by the tests (although the error message isn't very clear):
https://github.com/codespell-project/codespell/blob/bd79d997e62b87ef9131f5539e539f853b072ec5/codespell_lib/tests/test_dictionary.py#L118

Reply:
Message improved in #1493

Reply:
This is in now, so would you mind doing a Git update first @luzpaz before you fix this, so I can confirm it worked fine in Travis before you commit the fix?

Reply:
@peternewman result: https://travis-ci.org/github/codespell-project/codespell/jobs/698171480#L738  
```
E           AssertionError: 
E           error finisch: corrections "finish, finish," are not (lower-case) unique
```